src/potions.ts

```diff
@@ -476,10 +479,27 @@ export const rufusPotion = new Potion($item`closed-circuit pay phone`, {
 
     // We consider the average price of the shadow items to not get gated behind an expensive one
     const shadowItems = $items`shadow brick, shadow ice, shadow sinew, shadow glass, shadow stick, shadow skin, shadow flame, shadow fluid, shadow sausage, shadow bread, shadow venom, shadow nectar`;
-    const averagePrice =
-      sum(shadowItems, (it) =>
-        historical && historicalAge(it) < 14 ? historicalPrice(it) : mallPrice(it)
-      ) / shadowItems.length;
+    const getPrice = (item: Item): number =>
+      3 * (historical && historicalAge(item) < 14 ? historicalPrice(item) : mallPrice(item));
+    const avgSpeculativePrice = sum(shadowItems, (it) => getPrice(it)) / shadowItems.length;
+
+    if (haveItemQuest) return avgSpeculativePrice;
+
+    if (get("rufusDesiredItems") === "") {
+      directlyUse($item`closed-circuit pay phone`);
```

Comment:
`ClosedCircuitPayphone.chooseQuest`  lets you dynamically choose the quest using a function that takes, among other things, the Item it's looking for as an input; do we want to use that here, or does it not work?

Reply:
if it doesn't work for our purposes it doesn't work for our purposes, I just think it would be nice

Reply:
I do not want to choose a quest for this under any circumstances.

Reply:
To elaborate on the why, it would lock us into a quest which we can otherwise reroll the next day.